executor/builder.go

```diff
@@ -4664,3 +4667,30 @@ func (b *executorBuilder) validCanReadTemporaryTable(tbl *model.TableInfo) error
 
 	return nil
 }
+
+func (b *executorBuilder) readFromCache(tblInfo *model.TableInfo, startTS uint64) kv.MemBuffer {
+
```

Comment:
useless empty line

Reply:
sorry. i have delete it. but forget to push it.